app/src/main/java/woowacourse/shopping/data/repository/RemoteCatalogProductRepositoryImpl.kt

```
@@ -1,198 +1,78 @@
-                        }
-                    }
+    ): List<ProductUiModel> =
+        try {
+            remoteDataSource
+                .fetchProducts(
+                    category = category,
+                    page = page,
+                    size = size,
+                ).productContent
+                .map { it.toUiModel() }
+        } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)